--- raptor2-2.0.15/src/parsedate.c.omv~	2017-06-26 02:23:08.385276937 +0200
+++ raptor2-2.0.15/src/parsedate.c	2017-06-26 02:23:16.794080376 +0200
@@ -384,7 +384,7 @@ typedef short int yytype_int16;
 # define YYUSE(E) /* empty */
 #endif
 
-#if defined __GNUC__ && 407 <= __GNUC__ * 100 + __GNUC_MINOR__
+#if !defined(__clang__) && defined __GNUC__ && 407 <= __GNUC__ * 100 + __GNUC_MINOR__
 /* Suppress an incorrect diagnostic about yylval being uninitialized.  */
 # define YY_IGNORE_MAYBE_UNINITIALIZED_BEGIN \
     _Pragma ("GCC diagnostic push") \
--- raptor2-2.0.15/src/turtle_parser.c.omv~	2017-06-26 02:23:23.890914470 +0200
+++ raptor2-2.0.15/src/turtle_parser.c	2017-06-26 02:23:38.459573860 +0200
@@ -325,7 +325,7 @@ typedef short int yytype_int16;
 # define YYUSE(E) /* empty */
 #endif
 
-#if defined __GNUC__ && 407 <= __GNUC__ * 100 + __GNUC_MINOR__
+#if !defined(__clang__) && defined __GNUC__ && 407 <= __GNUC__ * 100 + __GNUC_MINOR__
 /* Suppress an incorrect diagnostic about yylval being uninitialized.  */
 # define YY_IGNORE_MAYBE_UNINITIALIZED_BEGIN \
     _Pragma ("GCC diagnostic push") \
