diff -up raptor2-2.0.16/src/parsedate.c.1~ raptor2-2.0.16/src/parsedate.c
--- raptor2-2.0.16/src/parsedate.c.1~	2024-07-08 14:56:49.353234768 +0200
+++ raptor2-2.0.16/src/parsedate.c	2024-07-08 14:58:56.534597080 +0200
@@ -449,7 +449,7 @@ typedef int yy_state_fast_t;
 #endif
 
 /* Suppress an incorrect diagnostic about yylval being uninitialized.  */
-#if defined __GNUC__ && ! defined __ICC && 406 <= __GNUC__ * 100 + __GNUC_MINOR__
+#if defined __GNUC__ && && ! defined __clang__ ! defined __ICC && 406 <= __GNUC__ * 100 + __GNUC_MINOR__
 # if __GNUC__ * 100 + __GNUC_MINOR__ < 407
 #  define YY_IGNORE_MAYBE_UNINITIALIZED_BEGIN                           \
     _Pragma ("GCC diagnostic push")                                     \
diff -up raptor2-2.0.16/src/turtle_parser.c.1~ raptor2-2.0.16/src/turtle_parser.c
